src/Microsoft.ML.TensorFlow/TensorFlow/Tensorflow.cs

```diff
@@ -700,6 +698,24 @@ public override string ToString()
             IntPtr len;
             return TF_GraphDebugString(Handle, out len);
         }
+
+        [DllImport(NativeBinding.TensorFlowLibrary)]
+        internal static extern string TF_OperationName(TF_Operation oper);
```

Comment:
Rather than directly exposing the TF C-API we should consider bringing back the TF# defined API (TFOperation class) on top of it.  See https://github.com/zeahmed/machinelearning/commit/b2a801600bb83883428cc0f0026b9249ad54b0b7#diff-ec7ea5716f3c05f773d3e1507b4f486aL729, https://github.com/zeahmed/machinelearning/commit/b2a801600bb83883428cc0f0026b9249ad54b0b7#diff-ec7ea5716f3c05f773d3e1507b4f486aL748.